index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="style.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"/>
+</head>
+
+<body>
+
+    <header class="navibar">
+        <p class="name">John Doe</p>
+        <img src="Images/hamburger.png" class="hamburger" alt="menu">
+    </header>
+    <div class="main">
```

Comment:
- [ ] Please make sure to use a `section` tag instead of a `div` in order to keep your code more semantic. Another option would be (only if you have different `section` tags) to use a `main` tag for the main content and the `section` tag for the rest of the page.